spring-cloud-contract-wiremock/src/main/java/org/springframework/cloud/contract/wiremock/WireMockTestExecutionListener.java

```diff
@@ -97,12 +93,7 @@ public void afterTestMethod(TestContext testContext) throws Exception {
 	}
 
 	private boolean annotationMissing(TestContext testContext) {
-		Annotation annotation = MergedAnnotations
-				.from(testContext.getTestClass(), MergedAnnotations.SearchStrategy.TYPE_HIERARCHY_AND_ENCLOSING_CLASSES,
-						RepeatableContainers.none())
-				.get(AutoConfigureWireMock.class).withNonMergedAttributes().synthesize(MergedAnnotation::isPresent)
-				.orElse(null);
-		if (annotation == null) {
+		if (TestContextAnnotationUtils.findMergedAnnotation(testContext.getTestClass(), AutoConfigureWireMock.class) == null) {
```

Comment:
As of Spring Framework 5.3.3, you should be able to replace that with `TestContextAnnotationUtils.hasAnnotation(testContext.getTestClass(), AutoConfigureWireMock.class)`.